events/events.go

```diff
@@ -7,7 +7,6 @@ package events
 
 import (
 	r3 "github.com/golang/geo/r3"
-
```

Comment:
I think I've finally found something for this :smile:.

`goreturns -local github.com/markus-wa/demoinfocs-golang events/events.go` makes it 'pretty' (in my eyes). I still don't know why it always breaks in the first place, I haven't actually configured anything like that but here's what you can do in VS Code:

`"go.formatTool": "goreturns"`
and
`"go.formatFlags": ["-local github.com/markus-wa/demoinfocs-golang"]`

Unfortunately that would mean you need different workspaces for the lib and your project - so I'm not sure if that's helpful if you don't already have that.